docker-compose.yml

```diff
@@ -1,5 +1,13 @@
 version: "3"
 services:
   mongo:
-    network_mode: host
```

Comment:
I like that you moved the host mode, I always forget that it doesnt work on mac or windows. Maybe just link up port 27017 so that its accessible on the host machine for mongo db tools.